app/build.gradle

```diff
@@ -48,22 +49,41 @@ android {
 
 final ANDROID_SUPPORT_VERSION = '25.1.1'
 final BUTTERKNIFE_VERSION = '8.5.1'
+final DART_VERSION = '2.0.2'
+final DB_FLOW_VERSION = '3.1.1'
```

Comment:
it has the UUID adapter and we always used a beta version and we've never had problems . BTW: it's a training project :smiley: